io/disk/Avago_storage_adapter/avago3008.py

```diff
@@ -92,42 +118,69 @@ def set_online_offline(self, state):
         if process.system(cmd, ignore_status=True, shell=True) != 0:
             self.fail("Failed to set drive to %s" % state)
 
-    def test_setoffline(self):
+    def adapter_status(self):
 
         """
-        This function set's a drive to offline
+        This is a helper function, to check the status of the adapter
         """
-        self.set_online_offline("offline")
+        cmd = "./sas3ircu %d status" % self.controller
+        if process.system(cmd, ignore_status=True, shell=True) != 0:
+            self.fail("Failed to display the status of the adapter")
```

Comment:
@vrbagalkote Guess it would be good to return `volume state`, which is `optimal` in this case